OutOfSchool/OutOfSchool.DataAccess/Models/SubDirection.cs

```diff
@@ -25,4 +26,6 @@ public class SubDirection : IKeyedEntity<long>, ISoftDeleted
     public virtual Direction Direction { get; set; }
 
     public virtual ICollection<InstitutionHierarchy> InstitutionHierarchies { get; set; }
+
+    public virtual List<CompetitiveEvent> CompetitiveEvents { get; set; } = [];
```

Comment:
i have not seen this nav property used yet, so unless we are going to use it - remove it. We will add it if we will need to access the relationship from this side.